src/panels/lovelace/cards/hui-glance-card.js

```
@@ -90,6 +96,11 @@ class HuiGlanceCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
   setConfig(config) {
     this._config = config;
     this.updateStyles({ '--glance-column-width': (config && config.column_width) || '20%' });
+
+    if (config && config.theming && typeof(config.theming) === 'string') {
```

If it's not a string, we should throw an exception.